farmdata2_modules/fd2_tabs/resources/CustomTableComponent.js

```diff
@@ -296,7 +296,7 @@ let CustomTableComponent = {
                 'id': this.rows[index].id,
                 'data': this.rows[index].data    
             }
-            this.$emit('edit-clicked')
+            this.$emit('edit-clicked', this.rows[index].data)

```

Comment:
This emits the full row data.  Does it make sense to emit just the `id` or is there value in emitting all of the data in the row?

Reply:
Either way, I believe it's a point of reference. If we emit the `id` here, then the parent page will have to take the time to find the log within its own rows array and grab that data there to determine what should be filtered. Would you like that to be something the parent page handles or the child component?

Reply:
I agree that since we have the data readily available it seems sensible to emit it so that the page doesn't have to look it up.  But on the other hand, there may be cases where the page will need to lookup the full log for something that doesn't appear in the `data`.   Is there something in `this.rows[index].data` that will allow the code in the page to look up the corresponding full log?  Or is that what the `id` does?

Reply:
In that case, it would make more sense to have it simply emit the `id` and have the parent page handle lookup of the log and grab whatever information it needs. Unfortunately the `this.rows[index].data` only emits that logs data without any way of quickly fetching that log from the rows array in the parent page. Not to mention even if we did workaround that issue two logs might have identical data but have different id's. So it is definitely better to use the `id` in this case and just allow the parent page to handle it. 

I didn't grab from `editedRowData` because I think that might cause problems down the line. Currently we want to populate a dropdown based on the seeding type of the log. Fortunately, seeding type is not an editable data property of a log, but it may be the case down the road where we want to populate some property and the data property we want to look at can be edited which might update the data property options. Is that something we want or do we want it to be set in stone once you hit the edit button?

Reply:
Agreed.  Let's just emit the `id` then.  The page can then look up the full log and obtain whatever data that it needs from there.